Makefile

```diff
@@ -1,7 +1,16 @@
-clean:
+clean: nondocs-clean docs-clean
+
+nondocs-clean:
 	rm -rf .benchmarks .eggs .tox .rally_it .cache build dist esrally.egg-info logs junit-py*.xml
+
+docs-clean:
 	cd docs && $(MAKE) clean
 
+# Avoid conflicts between .pyc/pycache related files created by local Python interpreters and other interpreters in Docker
+python-caches-clean:
```

Comment:
If I type `make python-caches-clean` locally (on MacOS), I get:

```
(.venv) daniel@io:Projects/rally ‹pr/487›$ make python-caches-clean
find ./ -name "__pycache__" -exec rm -rf -- \{\} \;
find: .//.venv/lib/python3.6/__pycache__: No such file or directory
[...]
find: .//tests/utils/__pycache__: No such file or directory
make: [python-caches-clean] Error 1 (ignored)
find ./ -name ".pyc" -exec rm -rf -- \{\} \;
```

Reply:
As discussed, this is harmless and should always return `exit 0` due to the `-` prefix in the command.

As discussed consecutive slashes will get collapsed as per the [Single Unix Specification](http://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap03.html#tag_03_266).

As discussed seems that using `.` instead of `./` makes the output look saner on macOS so I'll push this plus an `@` to suppress output altogether.

Reply:
Thanks. Makes sense.